endpoints-control/src/main/java/com/google/api/control/Client.java

```diff
@@ -162,13 +159,13 @@ public void stop() {
     Preconditions.checkState(running, "Cannot stop if it's not running");
 
     synchronized (this) {
-      log.log(Level.INFO, "stopping client background thread and flushing the report aggregator");
+      log.atInfo().log("stopping client background thread and flushing the report aggregator");
       for (ReportRequest req : reportAggregator.clear()) {
         try {
           transport.services().report(serviceName, req).execute();
         } catch (IOException e) {
-          log.log(Level.SEVERE,
-              String.format("direct send of a report request failed because of %s", e));
+          log.atSevere().withCause(e)
+              .log("direct send of a report request failed because of %s");
```

Comment:
There's no value for the `%s` placeholder in this one; the exception was being used as the value in the old version.